packages/block-editor/src/components/block-tools/insertion-point.js

```diff
@@ -168,6 +168,7 @@ function InbetweenInsertionPointPopover( {
 	// Other operations such as "group" are when the editor tries to create a row
 	// block by grouping the block being dragged with the block it's being dropped
 	// onto.
+	// @ZOOMOUTMODE
```

Comment:
Grouping while dragging onto things horizontally can be a fairly fine-grained action. For now, I lean toward leaving this in (return early if we're not inserting), but we could always revisit in follow-ups. I'll remove this comment for now.